TourMate/TourMate/Models/User.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  User.swift
+//  TourMate
+//
+//  Created by Keane Chan on 13/3/22.
+//
+
+import Foundation
+
+struct User {
+    let name: String
+    let email: String
+    let password: String
```

Comment:
Will we need to store password if Firebase Auth handles it for us?

Reply:
Yeah most likely not actually, I was a bit hesitant on this